bin/reth/src/cli/ext.rs

```diff
@@ -137,7 +137,7 @@ pub trait RethNodeCommandConfig: fmt::Debug {
     where
         Conf: PayloadBuilderConfig,
         Reth: RethNodeComponents,
-        Engine: EngineTypes + 'static,
+        Engine: EngineTypes<BuiltPayload = EthBuiltPayload> + 'static,
```

Comment:
once `build_empty_payload` returns `Self::BuiltPayload`, this bound should be removed